package.json

```
@@ -6,11 +6,10 @@
   "description": "Metadata API for Tezos-based tokens, supporting both FA1.2 and FA2 standards",
   "license": "MIT",
   "scripts": {
-    "start": "node src/index.js"
+    "start": "node --experimental-fetch src/index.js"
```

No need for this with Node.js v20. I guess I forgot to remove it

```suggestion
    "start": "node src/index.js"
```